src/biogeophys/InfiltrationExcessRunoffMod.F90

```diff
@@ -53,13 +53,22 @@ module InfiltrationExcessRunoffMod
      procedure, private :: InitCold
 
      procedure, private, nopass :: ComputeQinmaxHksat
-     procedure, private, nopass :: ComputeQinmaxVic
   end type infiltration_excess_runoff_type
 
   ! !PRIVATE DATA MEMBERS:
 
+  ! For methods that don't generate any infiltration excess runoff, we get this end result
+  ! by specifying a huge qinmax value - i.e., an effectively infinite max infiltration
+  ! rate.
+  !
+  ! 1e200 mm H2O /s seems large enough to be effectively infinite, while not being so
+  ! large as to cause floating point overflows elsewhere.
+  real(r8), parameter :: qinmax_unlimited = 1.e200_r8  ! mm H2O /s
```

Comment:
**Comment from @billsacks Sep 13, 2017**

When we talked about this a few weeks ago, there were suggestions of using a global constant here.

Reasons I went with a locally-defined qinmax_huge rather than a globally-defined constant:

- Globally-defined is useful to eliminate the need for, "If I change this here, then I also need to change this over there". That doesn't seem to be the case here:
  - Nobody needs to check this
  - I couldn't see any benefits to having this shared between different parameterizations
- Problems with globally-defined
  - A different value might make more sense in other contexts
  - If another parameterization needs a different value and so changes the global constant, it would change this and other previous parameterizations that use it, which could inadvertently change behavior

Reply:
**Comment from @martynpclark Sep 13, 2017**

Agreed. This seems like a good approach.